src/scenes/Share/Share.test.js

```diff
@@ -1,36 +1,43 @@
 import React from 'react';
-import { cleanup, render, wait, fireEvent } from '@testing-library/react';
+import { cleanup, render, screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
 import Share from './Share';
 import { SHARE_STATE, SHARE_PROVIDERS } from 'constants/sharing';
-import gsuite from './services/gsuite';
+import * as gsuite from './services/gsuite';
```

Comment:
@dmihalcik-virtru how does this change not break / isn't currently broken if we're not using `gsuite.defualt.init` below?